src/test/java/org/pesmypetcare/webservice/controller/appmanager/MyPetCareRestControllerTest.java

```diff
@@ -54,7 +52,7 @@ public void signUpShouldReturnStatusOK() throws Exception {
             .content(jsonUser1).param(key, password))
             .andExpect(status().isOk());
     }
-
+/*
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block comment has incorrect indentation level 0, expected is 4, indentation should be the same level as line 47.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5247402)